.github/workflows/test.yml

```diff
@@ -0,0 +1,21 @@
+name: pull-request
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+      - uses: actions/checkout@v2
+
+      - name: Set up node 
+        uses: actions/setup-node@v1
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Run tests 
+        run: yarn run test
```

Comment:
I had to make these changes to get it to run on CI:

```diff
-on:
-  pull_request:
-    branches: [ main ]
+on: [push, pull_request]

 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - name: Check out code
       - uses: actions/checkout@v2
-
-      - name: Set up node
-        uses: actions/setup-node@v1
-
       - name: Install dependencies
-        run: yarn install
-
-      - name: Run tests
+        # There's a bug in yarn 2 where checksums for packages built from a
+        # github repo have differing checksums depending on which platform
+        # they're built. Setting YARN_CHECKSUM_BEHAVIOR to `update` works
+        # around that problem for now.
+        #
+        # See: https://github.com/yarnpkg/berry/issues/2399 for more info.
+        run: YARN_CHECKSUM_BEHAVIOR=update yarn install --immutable
+      - name: Run tests
         run: yarn run test
diff --git a/package.json b/package.json
index a65f870..99abeb0 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,6 @@
 {
   "name": "workstreams-app",
   "version": "0.0.1",
-  "prepare": "svelte-kit sync",
   "scripts": {
     "dev": "vite dev",
     "build": "vite build",
@@ -12,7 +11,7 @@
     "lint": "prettier --ignore-path .gitignore --check --plugin-search-dir=. . && eslint --ignore-path .gitignore .",
     "format": "prettier --ignore-path .gitignore --write --plugin-search-dir=. .",
     "prepare": "husky install",
-    "postinstall": "scripts/install-twemoji-assets.sh && scripts/install-design-system-static-assets.sh",
+    "postinstall": "svelte-kit sync && scripts/install-twemoji-assets.sh && scripts/install-design-system-static-assets.sh",
     "apollo:update-schema": "npx apollo service:download --endpoint=https://api.thegraph.com/subgraphs/name/gh0stwheel/drips-on-ethereum graphql-schema.json",
     "typechain:generate-contract-types": "npx typechain --target ethers-v5 --out-dir './src/lib/stores/drips/contracts/types' './src/lib/stores/drips/contracts/**/*.json'",
     "apollo:generate-types": "npx apollo codegen:generate --localSchemaFile=graphql-schema.json --target=typescript --tagName=gql",
```

Reply:
@rudolfs are you sure it failed because of this? For some reason I see "every step must define a uses or run key" as the failure message. It looks as if the action is just completely blank before it gets merged to main, like it's not actually executing. Maybe I'm getting this wrong though.

I applied your diff either way 👍